ldap/data_source_resource_ldap_user.go

```diff
@@ -0,0 +1,80 @@
+package ldap
+
+import (
+	"github.com/Ouest-France/goldap"
+	"github.com/go-ldap/ldap/v3"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceLDAPUser() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceLDAPUserRead,
```

Comment:
With SDK v2, non context-aware functions are deprecated (https://www.terraform.io/docs/extend/guides/v2-upgrade-guide.html#more-support-for-context-context), so I just updated it too on group datasource/resource.